### PR TITLE
Add PHP 8.2 to testing workflow

### LINKED ISSUE #43: Run tests against PHP 8.2
Now that PHP 8.2 is released, we should run our test suite against it.

**Done is:**
* We've verified any PHP 8.2 compatibility issues are addressed.
* GitHub actions is updated to run the test suite against PHP 8.2.

### PR DESCRIPTION
**Pull request by wojsmol**
Fixes #43

### CODE REVIEW

File: .github/workflows/reusable-testing.yml [hunk 1]
```diff
@@ -89,6 +90,9 @@ jobs:
           - php: '8.1'
             wp: 'trunk'
             mysql: '8.0'
+          - php: '8.1'
```
Comment (danielbachhuber) [inline .github/workflows/reusable-testing.yml:93]:
```suggestion
          - php: '8.2'
```

Should this be 8.2?

**Review by danielbachhuber — COMMENTED** [review]
Thanks!

Out of curiosity, how does this impact the build status?

I'll defer to @schlessera on when we might want to land this. I think we'll want to have some bandwidth to fix any broken bits.

**Comment by wojsmol** [comment]
> Out of curiosity, how does this impact the build status?

As long as we have `continue-on-error: ${{ matrix.php == '8.2' }}` this will not fail the build https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

**Comment by schlessera** [comment]
Yes, this looks good. Running the tests on PHP 8.2 but still continuing on error will mean we don't fail the CI but still collect error info for PHP 8.2.

**Pull request merged**